client/dist/index.html

```diff
@@ -10,7 +10,7 @@
   <title>gainsboro_atelier</title>
 </head>
 <body>
-  <div id="app" style="display: flex; justify-content: center;"></div>
+  <div id="app" style="display: flex; flex-direction: column; justify-content: center;"></div>
```

Comment:
I think `style="display: flex; flex-direction: column; justify-content: center;"` will affect everyone's widget. Do you want to add this style to your local component instead of adding it to the top `app` component?